### PR TITLE
fix: set `recolor` to true

### PR DESCRIPTION
**Pull request by ti2811h**
* set the recolor option to true to recolor pages by default 

I was wondering why the pages where still white after I added the theme to my config and then I found out that you have to set the option recolor to true to recolor the pages.

### CODE REVIEW

**Comment by lokesh-krishna** [comment]
Looks good to me. What's the process around merging PRs presently?

**Review by sgoudham — APPROVED** [review]
Merging this as @lokesh-krishna has commented that it looks good.

In terms of PR merge process, we leave it up to the maintainers. Realistically it all depends on the type of PR and potentially the type of release process in the repository.

**Pull request merged**
